<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditPasswordDialog</class>
 <widget class="QDialog" name="EditPasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>417</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modification mot de passe</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Modification du mot de passe</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="usernameLayout">
       <item>
        <widget class="QLabel" name="usernameLabel">
         <property name="text">
          <string>Ancien mot de passe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="OldPasswordLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Entrer l'ancien mot de passe</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="OldPasswordcheckBox">
         <property name="text">
          <string>Voir</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="passwdLayout">
       <item>
        <widget class="QLabel" name="passwdLabel">
         <property name="text">
          <string>Nouveau mot de passe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NewPasswdLineEdit">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Entrer le nouveau mot de passe</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NewPasswordcheckBox">
         <property name="text">
          <string>Voir</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="passwdLayout_2">
       <item>
        <widget class="QLabel" name="passwdLabel_2">
         <property name="text">
          <string>Confirmation du nouveau mot de passe</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NewPasswdConfirmationLineEdit_2">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Entrer le nouveau mot de passe</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="NewConfirmPasswordcheckBox">
         <property name="text">
          <string>Voir</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonsLayout">
       <item>
        <widget class="QPushButton" name="SaveButton">
         <property name="text">
          <string>Sauvegarder</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>:/téléchargement.png</normaloff>:/téléchargement.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelButton">
         <property name="text">
          <string>Annuler</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
